.tekton/pipeline.yaml

```diff
@@ -0,0 +1,79 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: tektoncd-operator-pr
+spec:
+  params:
+    - name: repo_url
+    - name: revision
+  workspaces:
+  - name: source
+  tasks:
+    - name: fetch
+      taskRef:
+        name: git-clone
+      params:
+        - name: url
+          value: $(params.repo_url)
+        - name: revision
+          value: $(params.revision)
+      workspaces:
+      - name: output
+        workspace: source
+    - name: test
+      runAfter: [fetch]
+      taskRef:
+        name: golang-test
+      params:
+        - name: package
+          value: "github.com/openshift/tektoncd-operator"
+      workspaces:
+        - name: source
+          workspace: source
+    - name: lint
+      runAfter: [fetch]
+      taskRef:
+        name: golangci-lint
+      params:
+        - name: package
+          value: "."
+        - name: flags
+          value: "-v --timeout 15m"
+        - name: version
+          value: v1.24.0
+      workspaces:
+        - name: source
+          workspace: source
+  finally:
+    - name: finally
+      taskSpec:
+        steps:
+          - name: send-to-slack
+            env:
+              - name: SLACK_WEBHOOK_URL
+                valueFrom:
+                  secretKeyRef:
+                    name: slack-tektoncd-operator-ci-webhook
+                    key: hook_url
+              - name: PIPELINERUN
+                valueFrom:
+                  fieldRef:
+                    fieldPath: metadata.labels['tekton.dev/pipelineRun']
+              - name: GITHUB_PULL_LABEL
+                value: "{{pull_request.labels}}"
+              - name: LABEL_TO_CHECK
+                value: "nightly-ci"
+              - name: SUCCESS_URL_ICON
+                value: "https://github.com/tektoncd.png"
+              - name: FAILURE_URL_ICON
+                value: "https://www.vhv.rs/dpng/d/415-4154815_grumpy-cat-png-photos-grumpy-cat-png-transparent.png"
+              - name: SUCCESS_SUBJECT
```

Comment:
any chances to find some other icons url so we can easily make a difference with the pipelines-catalog ci ?

Reply:
yep, I can change those